src/iosrv/init.c

```diff
@@ -678,6 +757,9 @@ parse(int argc, char **argv)
 		case 'n':
 			dss_nvme_conf = optarg;
 			break;
+		case 'p':
+			numa_node = atoi(optarg);
```

Comment:
seems should be an optional string, to allow setting like: -p "1, 3", or -p "2-3", rather than a single integer. (can refer the "sub_tests" option in obj IO test).
if it is not set, then by default can use all cores.

The current patch will cause one server can only use cores on one numa node (if I read it correctly), and even when no this option specified it can only use the first numa node. I think this should be refined before landing.

Reply:
It is by design that it accepts a single integer, specifically to support the notion that it binds threads to a single NUMA node.